src/auth/activation.ts

```diff
@@ -57,5 +53,5 @@ async function showManageConnectionsOnStartup() {
     }
 
     // Show connection management to user
-    AuthCommandDeclarations.instance.declared.showManageConnections.execute('firstStartup')
+    showManageConnections.execute(placeholder, 'firstStartup')
```

Comment:
why can't we just pass undefined?

Reply:
Found out the hard way that in this edge case vscode can either pass an object (tree node command) OR undefined (ellipsis menu item)

Reply:
> undefined is a false positive that we didn't run in to this edge case.

why does that matter?